code/amber/python/lab03b_average_numbers.py

```diff
@@ -0,0 +1,55 @@
+#AVERAGE NUMBERS
+
+from curses.ascii import isalpha
+
+nums = [5, 0, 8, 3, 4, 1, 6]
+
+for num in nums:
```

Comment:
for loops are named as singular in plural to help visualize that the plural is a list, and the singular is a single item from the list.

In this case, nums are 5, 0, 8, 3, 4, 1, 6 so on the first loop num will be 5, then 0, then 8, and so on.

When you redefine num by doing num += num every loop this happens:

Loop 1:
Num is 5, 
Num is now 5 + 5 = 10

Loop 2: 
Num is 0
Num is now 0 + 0 = 0
 
and so on.

So instead of using num += num, you instead need to make a variable to keep your sum (here's a big hint in the name).